timon/webclient/webif1/src/components/Timon.vue

```diff
@@ -125,32 +134,35 @@ import axios from 'axios'
 var autorefreshInterval
 var THREESECONDS = 3000
 var notifyHist = {}
+var PROBE_STATUS_BY_PRIORITY = ["ERR", "TIM", "WAR", "UNK", "OK"]
 
 export default {
   name: 'Timon',
   data: function () {
     return {
-      lastUpd: Date.now(),
+      lastUpd: Date.now() / 1000,
```

Comment:
why do you change lastupd from milliseconds to seconds?

Is this realted to the other changes or just something else, that needed correcting?